### ISSUE TITLE
Application.Run leads to serious performance degradation

### ISSUE THREAD

**Issue by wimgielis** [IBM/paxapi#43]
Hello,

In the last couple of weeks I opened multiple tickets with IBM support regarding PAfE commands in regular Excel VBA modules.
Be it: UnlinkSheet, or RefreshSheet and similar functions.

These functions run fine in the end but whejn any of the preceding lines uses Application.Run towards regular TM1 functions, the same commands run very very slowly. Run time *5 or worse by just adding such a line. Examples:

- vElement = Application.Run("SUBNM", sTM1Server & ":" & sDimension, sSubsetToLoop, i)
- vElement_Code = Application.Run("DIMNM", sTM1Server & ":" & sDimension, app.Run("DIMIX", sTM1Server & ":" & sDimension, vElement))
- lNrOfVestigingen = Application.Run("SUBSIZ", sTM1Server & ":" & sDimension, sSubsetToLoop)

So basically within VBA I ask for the result of DIMIX, DIMNM, SUBSIZ but also DBRW on other occasions.

What could be the reason ? If I comment these lines in the code the other PAFE API commands run at normal speed.

Thanks,

Wim
